query07.sql

```diff
@@ -3,4 +3,13 @@
 */
 
 -- Enter your SQL query here
-select ...
+with q2_days as (
+SELECT start_time, end_time, EXTRACT(DAY FROM start_time) as start_day, EXTRACT (DAY FROM end_time) as end_day FROM public.indego_trips_2019_q2
+UNION ALL 
+SELECT start_time::date, end_time::date, EXTRACT(DAY FROM start_time::date) as start_day, EXTRACT (DAY FROM end_time::date) as end_day FROM public.indego_trips_2020_q2
+)
+select count (*)
+from q2_days
+where start_day != end_day
```

Comment:
Right track, but slightly over-counting, because some trip start and end times are more than one day apart (even though the max duration is minutes seconds). To really use this method, you would have to check for month boundaries as well (e.g. April 30 - May 1). The solution I used is [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L16-L25) (or a more compact equivalent solution [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L40-L42)). You could also [extract different parts from the date](https://www.postgresql.org/docs/9.1/functions-datetime.html#FUNCTIONS-DATETIME-EXTRACT), like `doy` (day of the year).